public/students/data.csv

```diff
@@ -0,0 +1,4 @@
+first_name,other_names,matric_number,gender,password
+Habib,Endurance,U16/FNS/MTH/1012,male,123456
+Tajudeen, Jimoh,U16/FNS/MTH/1017,male,123456
```

Comment:
I think its best to programmatically add the password than to expect the Lecturers to add it the csv. Dem fit f-up @bynalab

Reply:
Noted, will do.